src/styles/index.scss

```diff
@@ -1,3 +1,65 @@
+$days-of-week: (
+  mon: 0,
+  tue: 1,
+  wed: 2,
+  thu: 3,
+  fri: 4,
+  sat: 5,
+  sun: 6
+);
+
 body {
   margin: 0;
+  font-family: Arial, Helvetica, sans-serif;
+
 }
+
+.calendar {
+  width: 707px;
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1px;
+  &__day {
+    display: flex;
+    box-sizing: border-box;
+    height: 100px;
+    width: 100px;
+    background-color: #eee;
+    border: 1px solid black;
+
+    font-size: 30px;
+    align-items: center;
+    justify-content: center;
+    transition: 0.5s;
+  }
+
+  &__day:hover {
+    cursor: pointer;
+    background-color: #ffbfcb;
+    transform: translate(0, -20px);
+  }
+
+  @for $i from 28 through 31 {
+    &--month-length-#{$i} {
+    &__day:nth-child(n + #{$i + 1}) {
+        display: none;
+      }
+    }
+  }
+
+@each $day, $index in $days-of-week {
+  &--start-day-#{$day}
+  &__day:nth-child(1) {
+  margin-left: $index * 101px;
```

Comment:
The margin-left in `.calendar--start-day-#{$day} &__day:nth-child(1)` is derived from multiplying by 101px, which is a magic number. It's better to use variables or calculations based on the actual width and gap size to avoid hardcoding values.